packages/fxa-payments-server/src/routes/Subscriptions/PaymentUpdateForm.tsx

```diff
@@ -65,6 +93,68 @@ export const PaymentUpdateForm = ({
     resetUpdatePaymentIsSuccess,
   ]);
 
+  const {
+    exp_month,
+    exp_year,
+    payment_provider,
+    paypal_payment_error,
+  } = customer;
+
+  const actionButton = (
+    <ActionButton
+      {...{ customer, onRevealUpdateClick, revealFixPaymentModal }}
+    />
+  );
+
+  const errorAlertBarContent = (
```

Comment:
This displays over the header e.g. for a `'missing_agreement'` error; but that's a bug captured in #7903 .

Reply:
We also have https://jira.mozilla.com/browse/FXA-3229 logged for Phase 2 of Support Paypal